### DISCUSSION TITLE
How to enable android StrictMode?

### DISCUSSION THREAD

**Discussion by megajogos** [airsdk/Adobe-Runtime-Support#831] [Q&A]
We would like to enable android strictMode to detect ANR causes. How to do this with Harman Adobe Air?

https://developer.android.com/reference/android/os/StrictMode

**Comment by itlancer** [comment]
You are able to do that via Java native extension for your AIR application. There is no such instruments "out of the box" for AIR.

**Comment by megajogos** [comment]
Does anyone already have this ANE?

**Comment by marchbold** [comment]
I can put one together for you, just shoot me an email: airnativeextensions@distriqt.com
